doc/CITATION.cff

```diff
@@ -0,0 +1,35 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: icepyx
+message: >-
+  If you use this software, please cite the software version
+  you used by its version-specific DOI.
+  The DOI for each version is available on Zenodo (link below).
+type: software
+authors:
+  - family-names: The icepyx Developers
```

Comment:
Use 'name' instead of 'family-names', see https://github.com/citation-file-format/citation-file-format/blob/1.2.0/schema-guide.md#how-to-deal-with-unknown-individual-authors

```suggestion
  - name: The icepyx Developers
```